docs/_layouts/default.html

```diff
@@ -35,7 +35,7 @@
 			</a>
 			<a href="{{ site.baseurl }}/" alt="Hubble Enterprise">
 				<img src="{{ site.baseurl }}/assets/images/logo-title.svg" class="project-logo" width="3rem" height="3rem">
-				<h1 class="project-name"><script type="text/javascript">document.write(gheHostname())</script> Statistics</h1>
+				<h1 class="project-name">{{ site.url | remove:'https://' }} Statistics</h1>
```

Comment:
Could you plase add a space after the colon and put the `https://` part in double quotes for consistency with the rest?

Reply:
Yes, I have updated the file with your requested changes.